src/components/Form/Form.styled.jsx

```diff
@@ -60,8 +82,13 @@ export const FormButtonSecondary = styled.button`
     border: solid 1px var(--dark, #000);
     box-shadow: 2px 5px 6px rgb(0 0 0 / 0.3);
   }
-
   @media (min-width: 1025px) {
-    width: 200px;
+    width: 270px;
+  }
+  @media (min-width: 810px) {
+    width: 270px;
+  }
+  @media (min-width: 300px) {
+    width: 270px;
   }
```

Comment:
@angelik0828 what breakpoints do you suggest?

Reply:
> @angelik0828 what breakpoints do you suggest?

a maximum of 250 X 50 and a minimum of 200 X 40 can be used for the button